src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -177,18 +185,26 @@ private void AddStyle_SaveButton_Click(object sender, RoutedEventArgs e)
                 newItem.Name = "Input";
 
             //if the validation returns false it means that the new style that will be added doesn't exists
-            if (viewModel.ValidateExistingStyle(newItem) == false)
+            if (string.IsNullOrEmpty(groupNameLabel.Text))
             {
-                viewModel.AddStyle(newItem);
-                viewModel.ResetAddStyleControl();
-                AddStyleBorder.Visibility = Visibility.Collapsed;
-                AddStyleButton.IsEnabled = true;
+                viewModel.CurrentWarningMessage = Res.PreferencesViewEmptyStyleWarning;
+                viewModel.IsWarningEnabled = true;
+                viewModel.IsSaveButtonEnabled = false;
             }
-            else
+            else if (viewModel.ValidateExistingStyle(newItem) == true)
```

Comment:
The If condition here can be changed to if(viewModel.ValidateExistingStyle(newItem))

Reply:
@reddyashish updated in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12691/commits/59650714612e797d5bb8097f631f606a80ad15ea